python/cog/server/runner.py

```diff
@@ -232,13 +234,13 @@ def _run_prediction(
                     while True:
                         try:
                             self.predictor_pipe_writer.send(
-                                next(make_pickleable(output))
```

Comment:
Would this have worked if it were `make_pickleable(next(output))` instead? Feeling a little 🤦 about that!

Reply:
Hah, maybe? I think it returns the same iterator either way doesn't it.